mobile/v1/redshift/sql-runner/playbooks/standard/02-screen-views/README.md

```diff
@@ -0,0 +1,59 @@
+# Configuring the 02-screen-views playbooks
+
+The screen views module runs the standard mobile screen views model. It takes the `mobile_events_staged` table - produced by the Base module - as an input.
+
+`01-screen-views-main.yml.tmpl` runs the main mobile model logic. `99-screen-views-complete.yml.tmpl` runs the cleanup steps afterwards. `XX-destroy-screen-views.yml.tmpl` destroys all tables and manifests, for a complete rebuild.
+
+## Configuration quick reference
+
+### 01-screen-views-main
+
+`:input_schema:`       name of atomic schema
```

Comment:
Nit - for all of the playbook readmes, some of these lines are capitalised and some aren't.

Reply:
Nice catch. Changed